riskified-sdk/src/main/java/com/riskified/Utils.java

```diff
@@ -6,37 +6,36 @@
     public static final String SANDBOX_ENVIRONMENT = "https://sandbox.riskified.com";
     public static final String PRODUCTION_ENVIRONMENT = "https://wh.riskified.com";
     public static final String PRODUCTION_SYNC_ANALYZE_ENVIRONMENT = "https://wh-sync.riskified.com";
+    public static final String DECO_SANDBOX_ENVIRONMENT = "https://sandboxw.decopayments.com";
+    public static final String DECO_PRODUCTION_ENVIRONMENT = "https://w.decopayments.com";
 
     public static String getBaseUrlFromEnvironment(Environment environmentType) {
-        String url = null;
-        if (environmentType != null) {
-            switch (environmentType) {
-                case DEBUG:
-                    url = DEBUG_ENVIRONMENT;
-                    break;
-                case SANDBOX:
-                    url = SANDBOX_ENVIRONMENT;
-                    break;
-                case PRODUCTION:
-                    url = PRODUCTION_ENVIRONMENT;
-                    break;
-            }
-        }
+        String url = getUrlString(environmentType, SANDBOX_ENVIRONMENT, PRODUCTION_ENVIRONMENT);
         return url;
     }
     
-    public static String getBaseUrlSyncAnalzyeFromEnvironment(Environment environmentType) {
+    public static String getBaseUrlSyncAnalyzeFromEnvironment(Environment environmentType) {
+        String url = getUrlString(environmentType, SANDBOX_ENVIRONMENT, PRODUCTION_SYNC_ANALYZE_ENVIRONMENT);
+        return url;
+    }
+
+    public static String getDecoBaseFromEnvironment(Environment environmentType) {
+        String url = getUrlString(environmentType, DECO_SANDBOX_ENVIRONMENT, DECO_PRODUCTION_ENVIRONMENT);
```

Comment:
I guess you can just return and omit the temp variable assigning (same comment for the usages of 'getUrlString' in this class)

Reply:
👍